src/middlewared/middlewared/plugins/failover_/event.py

```diff
@@ -443,13 +435,18 @@ def vrrp_master(self, job, fobj, ifname, event):
                     vol['error'] = str(e)
                     failed.append(vol)
                     continue
+
+                # TODO: come back and fix this once we figure out how to properly manage zpool cachefile
+                # (i.e. we need a cachefile per zpool, and not a global one)
```

Comment:
What prevents us from implementing per-pool cache file right now?

Reply:
Time. This is currently blocking 22.12.2 release.